CS342MemberViews/Sources/JenMoore/GalleryView.swift

```diff
@@ -0,0 +1,97 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+public struct GalleryView: View {
+    @Binding var accentColor: Color
```

Comment:
You can remove the `@Binding` setting here as you do not need to pass any changes to the property back to your main views. Passing the constant value as a property to your subview will have the same effect and would optimize the times your view is redrawn.
You can learn more about a Binding here: https://developer.apple.com/documentation/swiftui/binding
```suggestion
    let accentColor: Color
```

Reply:
Oh, I see! Thank you for pointing this out!!